test/components/views/messages/__snapshots__/MLocationBody-test.tsx.snap

```diff
@@ -135,22 +148,12 @@ exports[`MLocationBody <MLocationBody> without error renders map correctly 1`] =
                             "lon": -0.1276,
                           },
                         ],
-                        Array [
```

Comment:
Setting the location is still tested some lines above.  
Without clearing the mocks it was duplicated.